tests/integration/components/side-test.js

```diff
@@ -13,7 +13,7 @@ test('tooltip-on-element shows on the top by default', function(assert) {
 
   assert.expect(1);
 
-  this.render(hbs`{{tooltip-on-element keepInWindow=false}}`);
+  this.render(hbs`{{tooltip-on-element keepInWindow=false effect='none'}}`);
```

Comment:
I'm not sure if it was this PR that broke it or if this test was actually passing but not actually doing what we wanted.